.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: cjdb
```

Comment:
I dont think you want to be assigned to every bug request :)

Reply:
Wow, good catch, you're absolutely right 😉 
Whom should I change this to (anyone at all?)